wa/commands/runstatus.py

```diff
@@ -0,0 +1,298 @@
+            output_path = args.directory
+            run_output = RunOutput(output_path)
+        else:
+            possible_outputs = list(discover_wa_outputs(os.getcwd()))
+            num_paths = len(possible_outputs)
+
+            if num_paths > 1:
+                print('More than one possible output directory found,'
+                      ' please choose a path from the following:'
+                      )
+
+                for i in range(num_paths):
+                    print(f"{i}: {possible_outputs[i].basepath}")
+
+                while True:
+                    try:
+                        select = int(input())
+                    except ValueError:
+                        print(f"Please select number from "
+                              f"{[x for x in range(num_paths)]}"
+                              )
+                        continue
+
+                    if select not in range(num_paths):
+                        print(f"Please select number from "
+                              f"{[x for x in range(num_paths)]}"
+                              )
+                        continue
+                    else:
+                        break
+
+                run_output = possible_outputs[select]
+
+            else:
+                run_output = possible_outputs[0]
+
+        rs = RunMonitor(run_output)
+        print(rs.generate_output(args.verbose))
+
+
+class RunMonitor:
+
+    @property
+    def elapsed_time(self):
+        if self._elapsed is None:
+            if self.ro.info.duration is None:
+                self._elapsed = datetime.utcnow() - self.ro.info.start_time
+            else:
+                self._elapsed = self.ro.info.duration
+        return self._elapsed
+
+    @property
+    def job_outputs(self):
+        if self._job_outputs is None:
+            self._job_outputs = {
+                (j_o.id, j_o.label, j_o.iteration): j_o for j_o in self.ro.jobs
+            }
+        return self._job_outputs
+
+    @property
+    def projected_duration(self):
+        elapsed = self.elapsed_time.total_seconds()
+        proj = timedelta(seconds=elapsed * (len(self.jobs) / len(self.segmented['finished'])))
+        return proj - self.elapsed_time
+
+    def __init__(self, ro):
+        self.ro = ro
+        self._elapsed = None
+        self._p_duration = None
+        self._job_outputs = None
+        self._termwidth = None
+        self.formatter = _simple_formatter()
+        self.get_data()
+
+    def get_data(self):
+        self.jobs = [state for label_id, state in self.ro.state.jobs.items()]
+        rc = self.ro._combined_config.run_config
+        self.segmented = segment_jobs_by_state(self.jobs,
+                                                rc.max_retries,
+                                                rc.retry_on_status
+                                                )
+
+    def generate_run_header(self):
+        info = self.ro.info
+        
+        header = ('\n=========================\n'
```

Comment:
Better to use multi-line string for this (using triple quotes` -- That will remove the need for `\n`'s

Reply:
Or alternatively may it could make use of the `underline` method?